modules/oc-mirror-updating-cluster-manifests.adoc

```diff
@@ -36,7 +36,11 @@ $ oc apply -f ./oc-mirror-workspace/results-1639608409/release-signatures/
 +
 [NOTE]
 ====
-If you are mirroring Operators instead of clusters, you do not need to run `$ oc apply -f ./oc-mirror-workspace/results-1639608409/release-signatures/`. Running that command will return an error, as there are no release image signatures to apply.
+* If you are using the oc-mirror plugin 4.15 or later and the OSUS Operator 5.0.3 or later to update your cluster in a disconnected environment, you can skip this step.
+However, the Cluster Version Operator must be configured to retrieve release image signatures from the OpenShift Update Service (OSUS).
+For more information, see "Configuring the Cluster Version Operator (CVO) to retrieve release image signatures" in the "Additional resources" section.
+
+* If you are mirroring Operators instead of clusters, you can skip this step. Running the previous command will return an error, as there are no release image signatures to apply.
```

Comment:
`If you are mirroring Operators instead of clusters` maybe made confusing, I think `If you are mirroring Operators instead of release images` or `If you are mirroring Operators instead of payload images` will be better, thought?

Reply:
Good catch, thanks! I think that was probably meant to say "cluster Operators" or something similar to your suggestions.

I can change it to say "release images" since that seems to make the most sense here.